pkgs/by-name/dp/dpp/package.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+, libopus
+, openssl
+, zlib
+, libsodium
+, gcc
+, pkg-config
+, autoPatchelfHook
+, lib
+}:
+stdenv.mkDerivation rec {
+  pname = "dpp";
+  version = "10.0.29";
+
+  src = fetchFromGitHub {
+    owner = "brainboxdotcc";
+    repo = "DPP";
+    rev = "v${version}";
+    hash = "sha256-BJMg3MLSfb9x/2lPHITeI3SWwW1OZVUUMVltTWUcw9I=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+  ] ++ lib.optional (stdenv.hostPlatform.isLinux) autoPatchelfHook;
+
+  buildInputs = [
+    openssl
+    zlib
+    libsodium
+    libopus
+  ];
+
+  meta = {
+    description = "Discord C++ Library";
+    homepage = "https://github.com/brainboxdotcc/DPP";
+    license = lib.licenses.asl20;
+    maintainers = with lib.maintainers; [ xbz ];
+  };
+}
```

Comment:
```suggestion
stdenv.mkDerivation (finalAttrs: {
  pname = "dpp";
  version = "10.0.29";

  src = fetchFromGitHub {
    owner = "brainboxdotcc";
    repo = "DPP";
    rev = "v${finalAttrs.version}";
    hash = "sha256-BJMg3MLSfb9x/2lPHITeI3SWwW1OZVUUMVltTWUcw9I=";
  };

  nativeBuildInputs = [
    cmake
    pkg-config
  ] ++ lib.optional (stdenv.hostPlatform.isLinux) autoPatchelfHook;

  buildInputs = [
    openssl
    zlib
    libsodium
    libopus
  ];

  meta = {
    description = "Discord C++ Library";
    homepage = "https://github.com/brainboxdotcc/DPP";
    license = lib.licenses.asl20;
    maintainers = with lib.maintainers; [ xbz ];
  };
})
```

Reply:
done, thank you for your input

Reply:
@Xbz-24 just as a tip, you can resolve the conversations you have already dealt with, this makes the thread cleaner.